test/e2e/framework/network/utils.go

```diff
@@ -1165,8 +1185,8 @@ func UnblockNetwork(ctx context.Context, from string, to string) {
 		return false, nil
 	})
 	if err != nil {
-		framework.Failf("Failed to remove the iptable REJECT rule. Manual intervention is "+
-			"required on host %s: remove rule %s, if exists", from, iptablesRule)
+		framework.Failf("Failed to remove the REJECT rule. Manual intervention is "+
+			"required on host %s: remove rule using command %s, if exists", from, undropCmd)
```

Comment:
"if exists" doesn't quite work there now that you switched from the rule to the command